metron-deployment/packaging/docker/rpm-docker/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+rm -rf SRPMS/ RPMS/ && \
+rpmbuild -v -ba --define "_topdir $(pwd)" --define "_version 0.2.0BETA" SPECS/metron.spec && \
+rpmlint -i SPECS/metron.spec RPMS/*/metron* SRPMS/metron
```

Comment:
Unless there's some risk that the rpmbuild command will fail silently or that the produced rpms won't work repeatable (or there's something else I'm missing), I'd drop the rpmlint command.